client/src/entrypoints/admin/core.js

```diff
@@ -244,13 +244,18 @@ $(() => {
   });
 
   /* Functions that need to run/rerun when active tabs are changed */
-  document.addEventListener('tab-changed', () => {
-    // Resize autosize textareas
+  function resizeTextAreas() {
```

Comment:
thought: this feels the right type of change to slowly move away from jQuery. Could this be accomplished with vanilla JS? 🤔

Reply:
Yep definitely could! Haha I sorta went backwards there 😅

Reply:
Ahh dang I could't refactor the Jquery portion to work properly. Maybe @thibaudcolas would have a suggestion, but it seems like theres some disconnect when I use `DOMContentLoaded` to resize on page load instead of jquery's `.ready()`